torch/csrc/dynamo/init.cpp

```diff
@@ -26,6 +27,12 @@ void initDynamoBindings(PyObject* torch) {
   if (guards == NULL || PyModule_AddObject(dynamo, "guards", guards) != 0) {
     throw python_error();
   }
+
+  PyObject* compiled_autograd = torch_c_dynamo_compiled_autograd_init();
+  if (compiled_autograd == NULL ||
```

Comment:
We are in C++17 world now, please use `nullptr` instead of C style `NULL`
```suggestion
  if (compiled_autograd == nullptr ||
```